dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -143,5 +143,11 @@
       }
     });
   }
+
+  // Export the functions which are provided by this file.
+  dashboard.dialog = {
+    showStartOverDialog: showStartOverDialog,
+    processResults: processResults
+  };
 })();
```

Comment:
If we're presenting this as an export, i wonder if the best approach might actually be something like:
```
window.dashboard = window.dashboard || {};
window.dashboard.dialog = (function () {
   // all of the logic here

   return {
    showStartOverDialog:    showStartOverDialog,
    processResults: processResults
  };
})();